src/controller/CHIPDeviceController.cpp

```diff
@@ -1886,9 +1886,12 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         {
             mPairingDelegate->OnCommissioningComplete(proxy->GetDeviceId(), CHIP_NO_ERROR);
         }
+        mCommissioningStage = CommissioningStage::kSecurePairing;
         break;
-    case CommissioningStage::kSecurePairing:
     case CommissioningStage::kError:
+        mCommissioningStage = CommissioningStage::kSecurePairing;
+        break;
+    case CommissioningStage::kSecurePairing:
```

Comment:
I have no way to evaluate this change, because none of these state enum values are documented....  What does `kSecurePairing` mean?

Reply:
It can be a bit misleading, but it indicates the initial state. It's set in the constructor and the method for starting commissioning requires that state.